# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/drm-fourcc/.cargo-checksum.json
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/drm-fourcc/.cargo-checksum.json b/third_party/rust/drm-fourcc/.cargo-checksum.json
deleted file mode 100644
--- a/third_party/rust/drm-fourcc/.cargo-checksum.json
+++ /dev/null
@@ -1,1 +0,0 @@
-{"files":{"Cargo.toml":"05f9a9cfba4c06202203421e396de95b0d6b2aacf6b88b64bd4dddcea92d9e86","README.md":"51afa4a71ce2aea69ed2c928b7abc6fcb6e361ba0f17b00c869456dd0a4de12f","build.rs":"d01ecd12113126de35dd710e840c651a4a6a1c72ef54233c44bab471a7838a46","src/as_enum.rs":"246f73c278aa45e3e7b10667c60d8d7399a43c52e9f1a05cf7c779747aa271ca","src/consts.rs":"0e542102e07b9df79b2edf3fdbb96e98b17cfb78383e56f4fabe37632654cc1f","src/lib.rs":"0c354a1f03eeae89c3022befde5249a19532d9eac2883673aa9397027f6abb51"},"package":"ebbf3a5ed4671aabffefce172ff43d69c1f27dd2c6aea28e5212a70f32ada0cf"}
\ No newline at end of file